raster/r.univar/r.univar_main.c

```diff
@@ -52,6 +57,14 @@ void set_params()
 	_("Percentile to calculate (requires extended statistics flag)");
     param.percentile->guisection = _("Extended");
     
+    param.threads = G_define_option();
+    param.threads->key = "nprocs";
```

Comment:
Should we add a "nproc" standard option for the parser (like e.g G_OPT_M_NPROCS)?
`nprocs` is used in several Python scripts as well. So having a standard option could secure a harmonized way to handle such an option...

Reply:
@ninsbl That's a good idea.

Reply:
Please check #1644.

Reply:
Right, I'll modify the code using the standard option.